<?xml version="1.0" ?>
<launch>

  <!--arg name="pointcloud_topic" default="/velodyne_points"/-->
  <arg name="elevation_map_topic" default="/elevation_mapping/elevation_map"/>
<param name="/pointcloud_topic" value="/D435_head_camera/depth/color/points" />

  <node name="plane_seg" pkg="plane_seg_ros" type="plane_seg_ros" output="screen">
    <remap from="/plane_seg/pose" to="/state_estimator/pose_in_odom"/>
    <param name="run_test_program" value="false" />
    <param name="pointcloud_topic" value="$(arg pointcloud_topic)" />
    <param name="elevation_map_topic" value="$(arg elevation_map_topic)" />
  </node>

</launch>
